### ISSUE TITLE
Peer dependencies missing with npm 3+

### ISSUE THREAD

**Issue by jbbr** [jedwards1211/meteor-webpack-react#50]
When using latest node 4.1.0 with npm 3.3 this package does not work anymore as it depends on peerDependencies of babel-core and others.

Most problematic: core-js is not available anymore in node_modules/babel-core/node_modules/core-js

I assume `core-js` has to be added as top level dependency and the `core-js-custom-build` has to be adjusted.

**Comment by ihistand** [comment]
Note that this issue also applies if npm was updated on a 12.7 nodejs. Doing a simple:
````
sudo npm update -g 
````
reproduces this issue.

**Comment by jbbr** [comment]
Have you tried applying my pull request #51 ?
With the fix npm 3 + npm 2 should work fine both

**Comment by ihistand** [comment]
@jbbr  I applied  #51  and it works as advertised.  I tested it as follows:

1 -  ran against the 0.12.7 node / 2.x npm.  Hello world works.
2 -  uninstalled node, removed all global node_modules. 
3 -  installed node 4.1  (built from source on the local Ubuntu box), and updated globally:
````
sudo npm update -g    # updated npm to 3.3.3
sudo npm install -g babel grunt-cli     # installed babel and grunt-cli globally, as per package recommendations
````
4 - cd to in the project directory, then: 
````
rm -rf node_modules
npm install
./dev
````
It all works!   Thanks for your PR!
--Ivan
